webapp/webapp/webapp/lib/load_config.py

```diff
@@ -75,7 +75,7 @@ def wait_for_db(self, app):
                 print('   '+str(len(tables))+' populated')
                 time.sleep(2)
         sysconfig = r.db('isard').table('config').get(1).run(conn)
-        if 'shares' in sysconfig.keys():
+        if sysconfig is not None and 'shares' in sysconfig.keys():
```

Comment:
Maybe the following code instead of `if` block is cleaner:
```
        app.shares_templates = sysconfig.get('shares', {}).get('templates', False)
        app.shares_isos = sysconfig.get('shares', {}).get('isos', False)
```

Reply:
Ok, pushed the new struct.
https://github.com/isard-vdi/isard/pull/256/commits/a931d5957fd5d47d8d3d8d5459ceaa95d59ff8ed